index.src.html

```diff
@@ -600,22 +603,21 @@ <h3 id="premission-prompt">Permission Prompt</h3>
   content checks. Migrating private network servers to HTTPS has indeed proven
   to be often difficult, even sometimes impossible.
 
-  We propose adding a new parameter to the `fetch()` options bag. For example:
+  A new parameter is added to the `fetch()` options bag:
   <pre highlight="js">
     fetch("http://router.local/ping", {
       targetAddressSpace: "private",
     });
   </pre>
+  This instructs the browser to allow the fetch even though the scheme is
+  non-secure and obtain a connection to the target server. The new `fetch()` API
+  is backward-compatible.
 
-  This would instruct the browser to allow the fetch even though the scheme is
-  non-secure and obtain a connection to the target server. This ensures the
-  feature cannot be abused to bypass mixed content in general.
-
+  This ensures the feature cannot be abused to bypass mixed content in general.
```

Comment:
Unclear what "this" refers to in the sentence. How about:

```suggestion
  Note that this feature cannot be abused to bypass mixed content in general.
```